lib/custom-logger.js

```diff
@@ -41,11 +41,7 @@ function pinoOptions(moduleOptions, options) {
 }
 
 function removePort(host) {
-  if (!host) {
-    return undefined
-  }
-  const hostname = host.split(':')
-  return hostname[0]
+  return !host ? undefined : host.split(':')[0]
```

Comment:
Why all in one line function? 
Maybe the hostname could be rewritten to be inline in return, but I would maintain the check separated. The code is quite more readable.